incubator/consul/templates/consul-petset.yaml

```diff
@@ -157,6 +157,12 @@ spec:
       - name: gossip-key
         secret:
           secretName: gossip-key
+      readinessProbe:
+        httpGet:
+          path: /v1/health/service/consul
```

Comment:
Okay thanks. I'll take a look at polling another endpoint. Do you have any suggestions?